docs/pom.xml

```diff
@@ -108,6 +108,8 @@
                         <surefire-version>${version.surefire.plugin}</surefire-version>
                         <graalvm-version>${graal-sdk.version-for-documentation}</graalvm-version>
                         <graalvm-flavor>${graal-sdk.version-for-documentation}-java11</graalvm-flavor>
+                        <mandrel-version>${mandrel.version-for-documentation}</mandrel-version>
+                        <mandrel-flavor>${mandrel.version-for-documentation}-java11</mandrel-flavor>
```

Comment:
Is this intentional? Wouldn't this expression, `${mandrel.version-for-documentation}-java11`, evaluate to `20.1.0.1.Alpha2-java11-java11`?

Reply:
Yeah, nicely caught!

Reply:
Pushed update with fix.